### PR TITLE
add luna tutorial

### PR DESCRIPTION
**Pull request by arfathpasha**
- replaces closed PR#249
- moved notebooks, conf, img to vmount
- this change simplifies tutorial and allows user to see updates made to notebooks and conf from host system without needing to create backups. There is a small risk of accidental commits of changed files to github but future integration testing can check for that. 
- updated paths in all confs for first three notebooks
- update paths in first three notebooks
- removec backup script
- updated readme
- tried alpine but decided ubuntu18 makes a better base because luna dependencies require native libs that alpine does not have and is hard to piece together
 
- noticed that dataset prep notebook writes to, 
    vmount/notebooks/computational\oncology_tmp_zips/ 
    vmount/notebooks/regional_bmps/ 
    vmount/notebooks/regional_npys/ 
    vmount/notebooks/slides  
    vmount/notebooks/dask-worker-space 
    vmount/notebooks/data-processing.log  
These outputs should go into project dir else it makes it hard to cleanup after a run

  - swapping pip install pyluna pyluna-pathology with pip install pyluna[pathology] didnt bring in the lib. import luna.pathology failed. Needs some debugging.

### CODE REVIEW

File: docker/luna_tutorial/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Luna Tutorial
+=============
+See make help for instructions on how to build and run the tutorial docker container. 
+
+```
+$ make help
+```
+
+In order to run this tutorial, first build the docker image and then lauch the container. Once the container is lauched, copy the url in the container's terminal log and paste it in a browser to open the jupyterlab environment containing the tutorial notebooks. Then step through the notebooks. Note that the url by default is set to localhost. If you need to run the tutorial from a remote system, replace the localhost IP address with the remote system IP address in the Makefile run target and rerun the container.
```
Comment (doori) [inline docker/luna_tutorial/README.md:9]: we could have a HOST or IP as an arg to make this edit easier.

File: docker/luna_tutorial/dockerfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ARG GROUP_ID
+ARG USER_NAME
+ARG GROUP_NAME
+RUN echo $USER_ID:$GROUP_ID $USER_NAME:$GROUP_NAME
+
+## install OS dependencies
+RUN apt-get update
+RUN apt-get install software-properties-common openslide-tools python-openslide default-jre git vim curl tree -y
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install python3.9 python3-pip -y
```
Comment (doori) [inline docker/luna_tutorial/dockerfile:16]: merging the RUN statements to 1 through out the dockerfile will reduce the number of layers during docker build.

File: docker/luna_tutorial/.dockerignore [hunk 3]
```diff
@@ -0,0 +1,7 @@
+*/*.svs
+*/*.tar
+*/*.parquet
+*/*.pt
+*/*.pil
+*/*.npy
+*/*.bmp
```
Comment (doori) [inline docker/luna_tutorial/.dockerignore:7]: Looks like `.ipynb_checkpoints` folders are checked in. Should this be added in .dockerignore

File: docker/luna_tutorial/README.md (hunk 1 above)
Comment (arfathpasha) [reply docker/luna_tutorial/README.md]: Good call! Added with localhost as default.

File: docker/luna_tutorial/dockerfile (hunk 2 above)
Comment (arfathpasha) [reply docker/luna_tutorial/dockerfile:16]: Good Call. Reduced layers.

File: docker/luna_tutorial/.dockerignore (hunk 3 above)
Comment (arfathpasha) [reply docker/luna_tutorial/.dockerignore:7]: Removed checkout dir from revision control.

File: docker/luna_tutorial/dockerfile [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+ENV PYSPARK_PYTHON="/usr/bin/python3"
+ENV PYSPARK_DRIVER_PYTHON="/usr/bin/python3"
+
+
+## verify installation
+RUN java --version
+RUN echo $JAVA_HOME
+RUN echo $LUNA_HOME
+RUN echo PYSPARK_PYTHON
+RUN echo PYSPARK_DRIVER_PYTHON
```
Comment (doori) [inline docker/luna_tutorial/dockerfile:53]: These will also add layers, may want to trim this down, or leave them as comments? if they are only here for debugging purposes. Missing $ in $PYSPARK_PYTHON $PYSPARK_DRIVER_PYTHON

File: docker/luna_tutorial/dockerfile (hunk 2 above)
Comment (doori) [reply docker/luna_tutorial/dockerfile:16]: Great! some of the apt-get could be merged too

Comment (doori) [reply docker/luna_tutorial/dockerfile:16]: Great! some of the apt-get could be merged too

**Review by doori — APPROVED** [review]
Great improvement simplifying the docker image and cleaning up the notebooks!

**Pull request merged**